src/forms/Checkbox.jsx

```diff
@@ -1,33 +1,24 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import cx from 'classnames';
-import Flex from '../layout/Flex';
-import FlexItem from '../layout/FlexItem';
-import Icon from '../media/Icon';
-
-export const FAUX_TOGGLE_CLASS = 'fauxToggle';
-export const FOCUSED_CHECKBOX_CLASS = 'focused';
-export const DISABLED_CHECKBOX_CLASS = 'disabled';
+import { default as SwarmCheckbox } from '@meetup/swarm-components/lib/Checkbox';
```

Comment:
These are exported from the root level as well so this could also be expressed as 
`import { Checkbox as SwarmCheckbox } from '@meetup/swarm-components'`

Reply:
That actually didn't work for me (pre-commit hook failed) in Select migration so I copied what Sadaf did here

Reply:
@sergeychernyshev let's pair to replicate this. I'm curious what the specific error was